src/components/OrderableList/FocusController.tsx

```diff
@@ -20,24 +20,29 @@ export const FocusController: FC<FocusControllerProps> = ({ children, width = Fo
 
         switch (event.code) {
             case "Space":
+                event.stopPropagation();
                 if (!isChildsEvent) {
+                    event.preventDefault();
                     childRef?.current?.focus();
                 }
                 break;
             case "Enter":
```

Comment:
You could combine this with the `"Space"` case if the actions are the same

Reply:
Updated: the Enter case required additional else logic :)